docs/UserGuide.md

```diff
@@ -394,7 +398,7 @@ Examples:
 
 Finds all appointments containing any of the specified keywords (case-insensitive) and/or with the given parameters. 
 
-Formats:
+Format:
 * `find appointment [n/NAME] [r/REMARKS] [d/DATE]* [t/TIME]*`
 * Note that all options marked with `*` are limited to one per query. 
```

Comment:
u mean one input? I think its unclear what one per query refers to

Reply:
@w-yuchen to fix this. I didn't touch any of their explanations, just fixed the ordering.